wom/src/main/scala-2.11/wom/executable/ExecutableValidation.scala

```diff
@@ -5,7 +5,7 @@ import common.Checked
 import common.validation.Checked._
 import common.validation.ErrorOr.ErrorOr
 import wom.callable.ExecutableCallable
-import wom.executable.Executable.{DelayedCoercionFunction, InputParsingFunction, ResolvedExecutableInputs}
+import wom.executable.Executable.{DelayedCoercionFunction, EGINInputIdentifier, InputParsingFunction, ResolvedExecutableInputs}
```

Comment:
ToL: should this (and others) be `egin` vs. `eGIN`?
- Example 1: [`ogin`](https://github.com/broadinstitute/cromwell/blob/1b1a56372659b9cb7a168bb1fa2a2296103e1256/wdl/src/main/scala/wdl/WdlCall.scala#L155)
- Example 2: [`wom`](https://github.com/broadinstitute/cromwell/blob/1b1a56372659b9cb7a168bb1fa2a2296103e1256/wom/src/main/scala/wom/values/WomValue.scala#L10)
- Example 3: [`ejea`](https://github.com/broadinstitute/cromwell/blob/1b1a56372659b9cb7a168bb1fa2a2296103e1256/engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaRunningJobSpec.scala#L14)
- etc.